tests/rating/kendo.rating.html

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>

```

Comment:
Please check the format of our tests - they need to live in JavaScript files only